keystore/keystore.go

```diff
@@ -202,14 +265,14 @@ func NewSCellKeyEncryptor(masterKey []byte) (*SCellKeyEncryptor, error) {
 }
 
 // Encrypt return encrypted key using masterKey and context.
-func (encryptor *SCellKeyEncryptor) Encrypt(key, context []byte) ([]byte, error) {
-	encrypted, _, err := encryptor.scell.Protect(key, context)
+func (encryptor *SCellKeyEncryptor) Encrypt(ctx context.Context, key []byte, keyContext KeyContext) ([]byte, error) {
+	encrypted, _, err := encryptor.scell.Protect(key, keyContext.Context)
```

Comment:
I think that previous scell encryptor should have helper that decides what data from the KeyContext to use for operations. Our current approach is to use ZoneID if specified otherwise ClientID. Much better will be recognize zones by ZoneID prefix otherwise use clientID. Not sure what priority should have Context for now